app/views/application/_subnavigation.html.erb

```diff
@@ -7,6 +7,7 @@
 
   <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-0 govuk-!-padding-right-0">
     <nav class="govuk-body govuk-!-margin-bottom-0 ">
+      <%= link_to "Organisations", organisations_path, class: active_tab(organisations_path) %>
```

Comment:
I don't think we want to show a link to this page. Only GDS are going to be able to get access to this page so we can just remember the url.

Reply:
👍 for this idea.